src/lib/utils/SynapseClient.ts

```diff
@@ -1208,22 +1209,22 @@ export const getUserTeamList = (
 }
 
 /**
- * Get the user's list of teams they are on
+ * Get a list of members for a team
  *
  * @param {*} id ownerID of the synapse user see -https://rest-docs.synapse.org/rest/GET/teamMembers/id.html
  * @param {*} fragment (optional) a prefix of the user's first or last name or email address (optional)
  * @param {*} limit    (optional) the maximum number of members to return (default 10, max limit 50)
  * @param {*} offset   (optional) the starting index of the returned results (default 0)
  *
  */
-export const getTeamList = (
+export const getTeamMembers = (
```

Comment:
This was incorrectly documented and is unused (for now -- I'll be using it in PORTALS-2316)